creator-node/src/apiHelpers.js

```diff
@@ -55,11 +56,16 @@ module.exports.handleResponseWithHeartbeat = (func) => {
       // Await the work of the endpoint
```

Comment:
can we delete heartbeat code now? (i think? @dmanjunath )

Reply:
we can't yet 🙁 . image_upload still uses it https://github.com/AudiusProject/audius-protocol/blob/10a85a817ef7184a1220332f827cd420ad364232/creator-node/src/routes/files.js#L641

Reply:
not yet 😞 image upload still uses it. but it's just a shift over to using the async processing queue. just another followup task